src/testnet/helium/node.rs

```diff
@@ -136,6 +137,17 @@ impl Node {
 
         let mut event_dispatcher = EventDispatcher::new();
 
+        // check for observer config in env vars
+        match std::env::var("STACKS_EVENT_OBSERVER") {
```

Comment:
I think it'd be better if this snippet lived in `config.rs`.

Reply:
Fixed